packages/arb-token-bridge-ui/src/token-bridge-sdk/cctp.ts

```diff
@@ -0,0 +1,165 @@
+import { readContract } from '@wagmi/core'
```

Comment:
Got rid of the useCctp hook and made it into a util function that can be exported through SDK. No change in logic.